CuPy installation found, continuing using GPU acceleration.
having the CPUs {0, 2, 4} available.
Started basinhopping at 2021-01-16 21:30:44.675154 UTC, with
command line arguments = {'path': '/home/jkgpu/HVQE/data/chain/periodic/own_graph/ops/energy/18', 'n_par': 28, 'par_multiplicity': 9, 'n_iter': 0, 'cost_fn': 'energy', 'temperature': 1.0, 'stepsize': 1.0, 'init_par': None, 'dump_interval': None}
runtime variables =  {'GPU': True, 'date_start': '2021-01-16 21:30:44.675154', 'complete_graph': [(1, 2), (3, 4), (5, 6), (7, 8), (9, 10), (11, 12), (13, 14), (15, 16), (17, 0), (0, 1), (2, 3), (4, 5), (6, 7), (8, 9), (10, 11), (12, 13), (14, 15), (16, 17)], 'init_layer': [(0, 1), (2, 3), (4, 5), (6, 7), (8, 9), (10, 11), (12, 13), (14, 15), (16, 17)], 'layers': [[(1, 2), (3, 4), (5, 6), (7, 8), (9, 10), (11, 12), (13, 14), (15, 16), (17, 0)], [(0, 1), (2, 3), (4, 5), (6, 7), (8, 9), (10, 11), (12, 13), (14, 15), (16, 17)]], 'n': 18}
........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................../home/jkgpu/miniconda3/lib/python3.8/site-packages/chainer/backends/cuda.py:142: UserWarning: cuDNN is not enabled.
Please reinstall CuPy after you install cudnn
(see https://docs-cupy.chainer.org/en/stable/install.html#install-cudnn).
  warnings.warn(
/home/jkgpu/miniconda3/lib/python3.8/site-packages/chainer/_environment_check.py:70: UserWarning: 
--------------------------------------------------------------------------------
CuPy (cupy) version 8.0.0 may not be compatible with this version of Chainer.
Please consider installing the supported version by running:
  $ pip install 'cupy>=6.3.0,<7.0.0'

See the following page for more details:
  https://docs-cupy.chainer.org/en/latest/install.html
--------------------------------------------------------------------------------

  warnings.warn(msg.format(
 
==========================================================================================
Finished basinhopping of  /home/jkgpu/HVQE/data/chain/periodic/own_graph/ops/energy/18 at 2021-01-16 22:50:56.533964 UTC, with
 
{'path': '/home/jkgpu/HVQE/data/chain/periodic/own_graph/ops/energy/18', 'n_par': 28, 'par_multiplicity': 9, 'n_iter': 0, 'cost_fn': 'energy', 'temperature': 1.0, 'stepsize': 1.0, 'init_par': None, 'dump_interval': None}
init_par = [0.00015494699634902, -0.00020058547037135588, 0.0002549383128459884, 0.00019081327831316545, 0.000363003422061866, -0.0004528189095065863, 4.8968681936956376e-05, -0.00013488091986189313, -0.000178838460894589, -2.6551492527202708e-05, -0.0003076125084179555, 0.00033277437167142084, -0.00030165179841947355, -0.00040249420247469346, -7.005180351947861e-05, -0.00021683951514060853, -0.00021190660872905955, -0.0002771512477825526, -9.354578334350161e-05, 0.00048685061029831236, -0.0001885533682189782, -0.00020229363012028834, -0.0003347604207797823, -1.3427400626603878e-05, -0.0002681812751932867, 0.00012029813974455135, -0.00018954736610145274, 0.00032870776140943054]
 
RESULTS:
--------
E_VQE = -8.022396223873171
inf_VQE = 0.00010807509743926413
n_fn_calls = 1130
Wall-clock time 1.3363880352841484 hours
 
For more output, see /home/jkgpu/HVQE/data/chain/periodic/own_graph/ops/energy/18/output.txt
Plots of output saved in /home/jkgpu/HVQE/data/chain/periodic/own_graph/ops/energy/18
End of HVQE.py
==========================================================================================
